bookwyrm/utils/regex.py

```diff
@@ -7,5 +7,6 @@
 STRICT_USERNAME = rf"(\B{STRICT_LOCALNAME}(@{DOMAIN})?\b)"
 FULL_USERNAME = rf"{LOCALNAME}@{DOMAIN}\b"
 SLUG = r"/s/(?P<slug>[-_a-z0-9]*)"
+HASHTAG = r"(#[^!@#$%^&*(),.?\":{}|<>\s]+)"
```

Comment:
Happy for better suggestions for this regex. I was looking at [Mastodon](https://github.com/mastodon/mastodon/blob/2889c686108e89a87317505f93b841f5a8a6649b/app/models/tag.rb#L29-L32)'s and [Twitter](https://github.com/twitter/twitter-text/blob/30e2430d90cff3b46393ea54caf511441983c260/rb/lib/twitter-text/regex.rb#L126-L165)'s and thought they might be slightly too complex for a first iteration.